test/specs/multiply.spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-len */
+/* jshint esversion: 6 */
+
+const Calculator = require('../../app/calculator');
+const {expect} = require('chai');
+
+describe(`positive tests for multiplication`, () => {
+  let calc;
+  beforeEach(() => {
+    calc = new Calculator();
+  });
+  afterEach(() => {
+    calc = null;
+  });
+
+  it(`shoud return summ of positive integer numbers`, () => {
+    expect(calc.multiply(4, 3, 6)).to.be.equal(4*3*6);
+  });
+
+  it(`shoud return summ of negative integer numbers`, () => {
```

Comment:
Does the `multiply()` method return a sum?

Reply:
fixed